CHANGELOG.md

```diff
@@ -9,10 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [`x.y.z`] - Unreleased
 ### Breaking changes:
+- `MaxRPCRingBufferSize` setting is removed. This was previously used to specify the RPC ring buffer size when generating schema. Now, `DefaultRPCRingBufferSize` is used, and can be overridden per RPC type using `RPCRingBufferSizeOverrides`.
 
 ### Features:
 - Added a message box notification when game is closed due to missing generated schema.
-- Adapted SpatialDebugger to use SubViews
+- Adapted SpatialDebugger to use SubViews.
+- Added a function flag `AlwaysWrite` that allows specifying an RPC to use a separate channel and allow overwriting unacked RPC calls. This is currently limited to Server Unreliable RPCs on classes inheriting from AActor, and only one such RPC can be specified per actor. This feature is disabled by default and can be enabled via `bEnableAlwaysWriteRPCs` setting.
```

Comment:
We're only intending on supporting this in 4.26 or later - worth adding a check that a user doesn't attempt to enable this in 4.25

Reply:
It's only applying this feature to movement RPCs that is 4.26+, since `ServerMovePacked` isn't a thing before then. I don't see any other benefit to disabling this feature completely prior to 4.26.